src/index.js

```diff
@@ -2,14 +2,26 @@ import express from 'express';
 import trimmer from 'trim-request-body';
 import cors from 'cors';
 import debug from 'debug';
+import swaggerUi from 'swagger-ui-express';
 import messages from './utils/messages';
 import response from './utils/response';
 import './config/env';
+import routes from './routes';
+import swaggerDoc from './config/swaggerDoc';
 
 // Instance of express app
 const app = express();
 
 const infoLog = debug('http:info');
+const router = express.Router();
+
+// Pass router to routes
+routes(router);
+
+// Pass router to swagger middleware
+swaggerDoc(router);
+
+app.use('/api', swaggerUi.serve, router);
```

Comment:
Let add version  one to the router here '/api/v1'

Reply:
Done